### PR TITLE
Create cluster-admin rolebinding

### PR DESCRIPTION
**Pull request by andraxylia**
Added instructions specific to GKE.

### CODE REVIEW

File: _docs/setup/kubernetes/quick-start.md [hunk 1]
```diff
@@ -28,24 +28,30 @@ If you wish to enable [transparent injection of sidecar]({{home}}/docs/setup/kub
...
     ```
-    Replace `<cluster-name>` with the name of the cluster you want to use and `<zone>`
-    with the zone that cluster is in.
+    where `<cluster-name>` is the name of the cluster you want to use, and `<zone>` is the zone that cluster is in.
 
-  * If you are using [IBM Bluemix Container Service](https://www.ibm.com/cloud-computing/bluemix/containers), retrieve your
-    credentials for kubectl:
+    2. Grant cluster admin permissions to the current user - admin permissions are required to create the necessary RBAC rules for Istio:
+    ```bash
+        kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user=$(gcloud config get-value core/account)
```
Comment (rshriram) [inline _docs/setup/kubernetes/quick-start.md]: this seems overly indented.. shouldn't it start from the same column as backtick?

Comment (andraxylia) [reply _docs/setup/kubernetes/quick-start.md]: yup, the indentation is wrong everywhere.

Comment (ldemailly) [reply _docs/setup/kubernetes/quick-start.md]: looks fine everywhere except here as far as I can see

**Comment by ldemailly** [comment]
merged so I can close
https://github.com/istio/issues/issues/4

**Pull request merged**